livre/eq/Config.cpp

```diff
@@ -131,6 +94,36 @@ const FrameData& Config::getFrameData() const
     return _impl->framedata;
 }
 
+::zerobuf::render::ImageJPEG& Config::getImageJPEG() const
+{
+#ifdef LIVRE_USE_ZEROEQ
+    return _impl->imageJPEG;
+#else
+    LBTHROW( std::runtime_error( "ZeroEq missing, impossible to send imageJPEG" ));
+#endif
+}
+
+bool Config::renderJPEG()
+{
+    _impl->imageJPEG.setData( std::vector< uint8_t >( ));
```

Comment:
Actually I think we can remove the #ifdef. imageJPEG is not a zeroEq but a 
zeroBuf class. It should work without ZeroEq via http ?